cluster/images/nginx/README.md

```diff
@@ -0,0 +1,8 @@
+### Nginx image
+This image is used to run nginx on the master.
+
+#### Instructions
+docker build -t gcr.io/google-containers/nginx:<version>
```

Comment:
@ArtfulCoder there are complexities around licensing and derived images I'd rather avoid.

@roberthbailey done.